src/index.css

```diff
@@ -36,6 +36,17 @@ header {
   width: 100%;
 }
 
+/* Radio buttons switch */
+
+input[type='radio'] { 
+  transform: scale(1.5); 
+  margin: 0 5px 0 5px;
+}
+
+#libraryLabel {
```

Comment:
This should be a class instead of an ID selector -> `.libraryLabel`.

Reply:
Fixed in: 91555c04c8088ee472c87cc25843642994ec92b4